### PR TITLE
Fix "Infiltration Sensible Heat Gain Energy" output

### LINKED ISSUE #9934: "Infiltration Sensible Heat Gain Energy" output wrong by orders of magnitude
Issue overview
--------------
There is a regression in E+ 23.1 where the "Infiltration Sensible Heat Gain Energy" (and similar outputs) are not being multiplied by the number of seconds in the timestep. As far as I can tell, it only affects the output variables, not simulation energy use.

### Details
Some additional details for this issue (if relevant):
 - Platform (Operating system, version)
 - Version of EnergyPlus (if using an intermediate build, include SHA)
 - Unmethours link or helpdesk ticket number

### Checklist
Add to this list or remove from it as applicable.  This is a simple templated set of guidelines.
 - [ ] Defect file added (list location of defect file here)
 - [ ] Ticket added to Pivotal for defect (development team task)
 - [ ] Pull request created (the pull request will have additional tasks related to reviewing changes that fix this defect)

### PR DESCRIPTION
**Pull request by shorowit**
Pull request overview
---------------------
 - Fixes #9934
 - Issue introduced in [this commit](https://github.com/NREL/EnergyPlus/commit/bc5300d04e509fa40a25c6f0bc306755f657fa8f)

**NOTE: ENHANCEMENTS MUST FOLLOW A SUBMISSION PROCESS INCLUDING A FEATURE PROPOSAL AND DESIGN DOCUMENT PRIOR TO SUBMITTING CODE**

### Pull Request Author
Add to this list or remove from it as applicable.  This is a simple templated set of guidelines.
 - [ ] Title of PR should be user-synopsis style (clearly understandable in a standalone changelog context)
 - [ ] Label the PR with at least one of: Defect, Refactoring, NewFeature, Performance, and/or DoNoPublish
 - [ ] Pull requests that impact EnergyPlus code must also include unit tests to cover enhancement or defect repair
 - [ ] Author should provide a "walkthrough" of relevant code changes using a GitHub code review comment process
 - [ ] If any diffs are expected, author must demonstrate they are justified using plots and descriptions
 - [ ] If changes fix a defect, the fix should be demonstrated in plots and descriptions
 - [ ] If any defect files are updated to a more recent version, upload new versions here or on DevSupport
 - [ ] If IDD requires transition, transition source, rules, ExpandObjects, and IDFs must be updated, and add IDDChange label
 - [ ] If structural output changes, add to output rules file and add OutputChange label
 - [ ] If adding/removing any LaTeX docs or figures, update that document's CMakeLists file dependencies

### Reviewer
This will not be exhaustively relevant to every PR.
 - [ ] Perform a Code Review on GitHub
 - [ ] If branch is behind develop, merge develop and build locally to check for side effects of the merge
 - [ ] If defect, verify by running develop branch and reproducing defect, then running PR and reproducing fix
 - [ ] If feature, test running new feature, try creative ways to break it
 - [ ] CI status: all green or justified
 - [ ] Check that performance is not impacted (CI Linux results include performance check)
 - [ ] Run Unit Test(s) locally
 - [ ] Check any new function arguments for performance impacts
 - [ ] Verify IDF naming conventions and styles, memos and notes and defaults
 - [ ] If new idf included, locally check the err file and other outputs

### CODE REVIEW

**Comment by shorowit** [comment]
Here's a test file to demonstrate:
[in.idf.txt](https://github.com/NREL/EnergyPlus/files/11142523/in.idf.txt)
[in.epw.txt](https://github.com/NREL/EnergyPlus/files/11142525/in.epw.txt)

Max timeseries value for Infiltration Sensible Heat Gain Energy:
- Before: 1329.21 J
- After: 4785156 J

Value is now greater by a factor of 3600, which is the expected difference because the simulation has an hour timestep.

File: src/EnergyPlus/HVACManager.cc [hunk 1]
```diff
@@ -2397,7 +2397,7 @@ void ReportInfiltrations(EnergyPlusData &state)
         } else if (thisZoneHB.MAT <= thisZone.OutDryBulbTemp) {
 
             thisInfiltration.InfilHeatGain =
-                thisInfiltration.MCpI_temp * (thisZone.OutDryBulbTemp - thisZoneHB.MAT) * TimeStepSys * ADSCorrectionFactor;
+                thisInfiltration.MCpI_temp * (thisZone.OutDryBulbTemp - thisZoneHB.MAT) * TimeStepSysSec * ADSCorrectionFactor;
```
Comment (shorowit) [inline src/EnergyPlus/HVACManager.cc:2400]: If you look at the `if` statement above, or the [commit where the bug was introduced](https://github.com/NREL/EnergyPlus/commit/bc5300d04e509fa40a25c6f0bc306755f657fa8f), it should be pretty obvious that this is a correct change.

Comment (Myoldmopar) [reply src/EnergyPlus/HVACManager.cc:2400]: Yeah, this is an obvious change both based on the context and the original commit that changed it.  Thanks.

**Review by Myoldmopar — APPROVED** [review]

**Comment by Myoldmopar** [comment]
Thanks @shorowit , merging.

**Pull request merged**